components/TeamPage/TeamPage.js

```diff
@@ -0,0 +1,89 @@
+import './TeamPage.css';
+import TeamMember from './TeamMember';
+import React, {useState, useEffect} from 'react';
+import Constants from '../../global_constants/Constants';
+
+/**
+ * @returns {React.Component}
+ */
+const TeamPage = () => {
+    const [teamMembers, setTeamMembers] = useState([]);
+    const [filter, setFilter] = useState('All');
+    let data = require('./team_members.json');
+    
+    
+    // const fetchData = () => {
+    //     fetch('https://jsonplaceholder.typicode.com/posts')
+    //       .then(response => {
+    //         return response.json();
+    //       })
+    //       .then(people => {
+    //         setLoadedPeople(people);
+    //       })
+    //       .catch(err => {
+    //         console.log(err);
+    //       });
+    // };
+
+    useEffect(() => {
+        //fetchData();
+        setTeamMembers(data);
+      }, []);
+
+    return <>
+        <h1 className="header">Meet the Team</h1>
+        <body>
+            <div className="teams">
+                <seciton className="col-xs">
```

Comment:
`seciton`

also we can probably use an array mapping here and abstract this to another class